spec/datadog/tracing/contrib/aws/service/states_spec.rb

```diff
@@ -12,28 +12,44 @@
   end
 
   context 'with execution_arn provided' do
-    let(:execution_arn) { 'arn:aws:states:us-east-1:123456789012:execution:example-state-machine:example-execution' }
+    let(:execution_arn) { 'arn:aws:states:sa-east-1:123456789012:express:targetStateMachineName:1234:5678' }
     let(:params) { { execution_arn: execution_arn } }
 
-    it 'sets the state_machine_name based on the execution_arn' do
+    it 'sets the state_machine_name, execution_arn, and aws_account based on the execution_arn' do
```

Comment:
this is failing in ci https://app.circleci.com/pipelines/github/DataDog/dd-trace-rb/12117/workflows/d71d2fd8-c667-44a6-8d36-25bc9ff3e1ed/jobs/455712/tests

Reply:
It looks like you found a fix for this, correct?

Reply:
yes

Reply:
simple typo